packages/ethereum-contracts/test/contracts/agreements/ConstantFlowAgreementV1.behavior.js

```diff
@@ -219,9 +224,15 @@ async function _shouldChangeFlow({
             break;
         case "createFlowByOperator":
         case "updateFlowByOperator":
-            tx = await testenv.contracts.superfluid.callAgreement(
+            cfa = await testenv.sf.contracts.IConstantFlowAgreementV1.at(
```

Comment:
follow up: we get the truffle contracts here again (instead of the ethers ones initialized in testenvironment) in order to be able to expect the events